examples/src/Viewer.tsx

```diff
@@ -391,11 +392,19 @@ class Viewer extends React.Component<InputParams, ViewerState> {
     }
 
     public convertFile(obj: Record<string, any>, fileType: TrajectoryType) {
+        const fileName = uuidv4() + ".simularium";
         simulariumController
-            .convertAndLoadTrajectory(this.netConnectionSettings, obj, fileType)
+            .convertTrajectory(this.netConnectionSettings, obj, fileType, fileName)
             .then(() => {
                 this.clearPendingFile();
             })
+            .then(() => {
+                simulariumController.changeFile(
+                    { netConnectionSettings: this.netConnectionSettings, },
+                    fileName,
+                    true,
+                )
```

Comment:
Understanding question, does this switch the viewer over to the new file?

Reply:
yes, this call does the bulk of the work of clearing the previous file, and requests the first frame for the new file

Reply:
Just checking my understanding.  Is it possible to convertFile, and then load a different file, and then when conversion is done, suddenly the viewer will switch over to the newly converted file?